ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioDev.xcscheme

```diff
@@ -125,6 +124,16 @@
       savedToolIdentifier = ""
       useCustomWorkingDirectory = "NO"
       debugDocumentVersioning = "YES">
+      <BuildableProductRunnable
+         runnableDebuggingMode = "0">
+         <BuildableReference
+            BuildableIdentifier = "primary"
+            BlueprintIdentifier = "889CAB5E27D46CD6007F5892"
+            BuildableName = "HelioDev.app"
+            BlueprintName = "HelioDev"
+            ReferencedContainer = "container:Helio.xcodeproj">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The app name and references have been updated from "BetterSocialDev" to "HelioDev". This is consistent with the PR title and seems to be an expected change.
```diff
- BuildableName = "BetterSocialDev.app"
- BlueprintName = "BetterSocialDev"
- ReferencedContainer = "container:BetterSocial.xcodeproj"
+ BuildableName = "HelioDev.app"
+ BlueprintName = "HelioDev"
+ ReferencedContainer = "container:Helio.xcodeproj"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->